service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/LanguageRequirementsCheckerTest.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.model.tasklist.TaskState.COMPLETED_FINISHED;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = LanguageRequirementsChecker.class)
+class LanguageRequirementsCheckerTest {
+
+    @Autowired
+    private LanguageRequirementsChecker underTest;
+
+    private static final CaseData ANY_CASE_DATA = CaseData
```

Comment:
could be just local to the test it is used in or you can replace the one in `shouldReturnEmptyErrorsAndCompletedState` with this as well

Reply:
my bad, forgot to clean that test up before pushing up

Reply:
Updated